sdk/dapp-kit/src/hooks/wallet/useSignAndExecuteTransactionBlock.ts

```diff
@@ -74,8 +78,13 @@ export function useSignAndExecuteTransactionBlock({
 				);
 			}
 
-			if (executeFromWallet) {
-				const walletFeature = currentWallet.features['sui:signAndExecuteTransactionBlock'];
+			const shouldExecuteFromWallet =
```

Comment:
Can we just make this _always_ execute from the wallet and make you use two hooks if you want it otherwise?

Reply:
We can remove the option, but this currently supports wallets that only support signing.  For example the zksend wallet doesn't support execution